CONTRIBUTING.md

```diff
@@ -1,3 +1,43 @@
-# Contributing an Instant Answer
+# Contributing an Instant Answer to DuckDuckGo.com
 
-An overview of how to submit an Instant Answer can [be found over in our docs](http://docs.duckduckhack.com/submitting/submitting-overview.html).
\ No newline at end of file
+> You can find the full [DuckDuckHack documentation here](http://docs.duckduckhack.com)
+
+Have an idea [that no one has done yet](https://duck.co/ia)? Improving an [existing Instant Answer](https://duck.co/ia/dev/issues)? 
+
+1. **Before coding, [create an Instant Answer Page](https://duck.co/ia/new_ia) for your idea.** If you're fixing an existing Instant Answer, [locate its existing page](https://duck.co/ia).
+
+	> Instant Answer Pages are the "home base" for each Instant Answer. They are where the community plans, collaborates, and tracks the [life cycle](http://docs.duckduckhack.com/submitting/long-term.html) of all Instant Answers. It's where we aggregate pull requests, issues, attribution, and feedback.
+
+2. **Hack away, keeping the [Instant Answer Production Guidelines](http://docs.duckduckhack.com/submitting/checklist.html) in mind.**
+
+3. **[Submit a pull request](http://docs.duckduckhack.com/submitting/pull-request.html).** 
+
+	> Make sure to paste your Instant Answer page URL in the description. This will automatically link the two.
+
+We're excited to meet you and support you along the way - and it's never too early to say hello. Join us on [Slack](mailto:QuackSlack@duckduckgo.com?subject=AddMe) or [email](mailto:open@duckduckgo.com). 
+
+*The opportunities to contribute don't stop here. Learn about [keeping your IA awesome](http://docs.duckduckhack.com/submitting/long-term.html).*
+
+
+## Pull Request Format
+
+When submitting a pull request, the following guidelines help speed up the review process:
+
+If your IA is new, use the format: `"New {IA TOPIC} {IA TYPE}"`. For example:
+
+	- "New Instagram Spice"
+	- "New Firefox Cheat Sheet"
+	- "New Color Hex Goodie"
+	
+If you're submitting a fix, use the format: `"{IA NAME}: Fixes #ISSUE" - {Explanation}` as the title (Conveniently, this syntax will auto-close the Github issue when your pull request is merged.). For example:
```

Comment:
Just wondering if it's better to put the `Fixes #ISSUE` in the description rather than in the title? Same goes for Goodies IA (duckduckgo/zeroclickinfo-goodies#2090).

It might get a bit confusing to see 2 numbers like this:
<img width="710" alt="screen shot 2016-01-24 at 8 16 00 pm" src="https://cloud.githubusercontent.com/assets/5793364/12536159/6f1f5d4c-c2d7-11e5-95e4-39307f7e159a.png">